src/Ratchet/Wamp/ServerProtocol.php

```diff
@@ -93,6 +93,10 @@ public function onMessage(ConnectionInterface $from, $msg) {
             throw new Exception("Invalid WAMP message format");
         }
 
+        if (isset($json[1]) && !is_string($json[1])) {
```

Comment:
I wasn't able to locate the spec, but is an integer a valid topic?

Reply:
The spec actually says full URLs but I've just ignored that.

Reply:
Although, even though this is now enforcing the spec this probably constitutes as a BC break. I think I'll update this to accept an int.